mobile/src/main/java/ch/epfl/sdp/mobile/application/chess/ChessFacade.kt

```diff
@@ -10,6 +10,8 @@ import ch.epfl.sdp.mobile.application.chess.notation.serialize
 import ch.epfl.sdp.mobile.application.toProfile
 import ch.epfl.sdp.mobile.infrastructure.persistence.auth.Auth
 import ch.epfl.sdp.mobile.infrastructure.persistence.store.*
+import ch.epfl.sdp.mobile.ui.social.*
+import java.util.*
```

Comment:
The imports should not be changed unless some new code actually uses them.

Reply:
Fixed in 87b7aea